torch/csrc/jit/serialization/export.cpp

```diff
@@ -220,6 +220,14 @@ class EncoderBase {
       bool use_external_data_format = false,
       const std::string& onnx_file_path = std::string());
 
+  void AddInitializersIntoGraphProto(
```

Comment:
Thanks @fatcat-z, the change here looks good to me.
Could you just a another test for a script module in addition to this trace module test?
You can just convert this model with: 
script_model = torch.jit.script(test_model)

And then test it similarly.

Reply:
Has added one test for script module.